ui/apps/platform/src/Containers/MainPage/Navigation/HorizontalSubnav.tsx

```diff
@@ -78,6 +79,14 @@ function getSubnavDescriptionGroups(
                               path: vulnerabilitiesInactiveImagesPath,
                               routeKey: 'vulnerabilities/inactive-images',
                           },
+                          {
+                              type: 'link',
+                              content: 'Images without CVEs',
+                              description:
+                                  'Images and workloads without observed CVEs (results might include false negatives due to scanner limitations, such as unsupported operating systems)',
```

Comment:
Images **and workloads** makes me pause.
* User **workloads** are determined by namespaces of deployments, correct?
* Remind me, will this view include images without CVEs from platform deployments?

In case either or both of us need a reason to smile, I did enjoy a charming redundancy at the end of bus rides with Greyhound: “This is your last and final destination.”

Reply:
> User workloads are determined by namespaces of deployments, correct?

Yes, correct.

> Remind me, will this view include images without CVEs from platform deployments?

Yes, this will also include images without CVEs from platform deployments. I wonder if that is worth including explicitly in the text here, similar to "All images" above?